app/views/dishes/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>Dish Show Page</h1>
+<h2><%= @dish.name %></h2>
+<h3>Ingredients</h3>
+ <% @dish.ingredients.each do |ingredient| %>
+  <p><%= ingredient.name %></p>
+<% end %>
+<h3>Total Calories</h3>
+<p><%= @dish.calory_count %>
+<h3><%= Chef %></h3>
+<p><%= @dish.chef.name %></p>
+<br>
+<%= form_with url: "/dishes/#{@dish.id}", method: :patch, data: {turbo: false} do |f| %>
+  <%= f.label "Add Ingredient" %>
+  <%= f.text_field :ingredient_id %>
+  <%= f.submit "Submit" %>
+<% end %>
```

Comment:
While this form is what I would consider RESTful, it is not RESTful in what it's aiming to accomplish. The task we are given is to associate a new ingredient to a dish, this does not happen on the dish resource, this is a new relationship being CREATED on the DISH_INGREDIENTS table between a dish and an ingredient. We are not updating anything on the dish table itself, therefore a `PATCH` to "/dishes/#{@dish.id}" is incorrect.

Therefore, your form should be making a `POST` request to `"/dishes/#{@dish.id}/ingredients"`.

```erb
<%= form_with url: "/dishes/#{@dish.id}/ingredients", method: :post, local: true, data: {turbo: false} do |f| %>
  <%= f.label "Add Ingredient" %>
  <%= f.number_field :ingredient_id %>
  <%= f.submit "Submit" %>
<% end %>
```